docs/src/tutorials/software-management.md

```diff
@@ -0,0 +1,120 @@
+
+The `tedge connect` will automatically start and enable the software management services.
+Find more about [how to connect thin-edge device to cloud](../howto-guides/004_connect.md)
+
+Once the thin-edge device is successfully connected to Cumulocity cloud, the **Software** option will be enabled and
+the list of softwares that are installed on the device will be visible as shown in the figure below.
+
+![Add new software](./images/start-software-management.png)
+
+
+> Note: Disconnecting thin-edge device from cloud will stop and disable the software management services.
+
+
+
+## Managing the device software on Cumulocity
+
+More info about managing the device software can be found [here](https://cumulocity.com/guides/users-guide/device-management/#managing-device-software)
+
+### Adding new software into software repository
+
+1. In the Software repository page, click Add software at the right of the top menu bar.
+2. In the resulting dialog box,
+   - to add a new software, enter a name for the software (and confirm it by clicking Create new in the resulting window),
+     a description and its version.
+   - to add a new version, select the software for which you want to add a new version from the dropdown list in the Software
+     field and enter a version and plugin type as **version::apt** (In the picture below the version is empty and are optional)
+3. Optionally, you can define the device type filter when adding a new software.
+4. As of now thin-edge.io only supports software from apt repositories. So, select the **Provide a file path file path** option and give an **empty space**.
+5. Click Save.
+
+Follow above mentioned steps as shown in the diagram
+
+ ![Add new software](./images/add-new-software-to-repo.png)
+
+### Deleting software or software version
+
+Find more information about how to delete the software or the specific software version [here](https://cumulocity.com/guides/users-guide/device-management/#deleting-softwares-or-software-versions)
+
+
+
+## Managing software on a device
+
+Find more information about [how to manage the software](https://cumulocity.com/guides/users-guide/device-management/#managing-software-on-a-device) on a device.
+
+### Install software on a device
+Find about how to [Install a specific software on a device](https://cumulocity.com/guides/users-guide/device-management/#to-install-software-on-a-device)
+> Note: Software profiles are not supported as of now on thin-edge.
+
+### Update software on a device
+Find about how to [Update the software on a device](https://cumulocity.com/guides/users-guide/device-management/#to-update-software-on-a-device)
+
+### Remove software on a device
+Hover over the software entry which you want to remove and click the **Remove** software icon and click on `Apply changes` button.
+
+
+
+## Starting and Stopping the Software management services manually
+
+For debugging purpose the services can be started/stopped manually as shown below.
```

Comment:
Not just for debugging purposes, but if someone wants to disable the software management feature and re-enable it later, then also these commands would be useful.